### ISSUE TITLE
Bug report Ollama Socks Issue with remote ollama instance

### ISSUE THREAD

**Issue by Hempflingclub** [your-papa/obsidian-Smart2Brain#63]
### What happened?

![grafik](https://github.com/your-papa/obsidian-Smart2Brain/assets/29497373/51a1ecc7-a6b8-4451-9156-6d67b15a8454)

![grafik](https://github.com/your-papa/obsidian-Smart2Brain/assets/29497373/0f0e79bb-67f2-491b-a322-f852b96b4232)

### Error Statement

[S2B] Ollama is not running Error: net::ERR_SOCKS_CONNECTION_FAILED
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)
    at SimpleURLLoaderWrapper.emit (node:events:513:28)

### Smart Second Brain Version

0.6.0

### Debug Info

SYSTEM INFO:
	Obsidian version: v1.5.11
	Installer version: v1.5.3
	Operating system: #1 SMP PREEMPT_DYNAMIC Debian 6.1.76-1 (2024-02-01) 6.1.0-18-amd64
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Things v2.1.19
	Snippets enabled: 1
	Restricted mode: off
	Plugins installed: 20
	Plugins enabled: 17
		1: Advanced Tables v0.21.0
		2: Excalidraw v2.0.25
		3: Dataview v0.5.66
		4: Full Calendar v0.10.7
		5: Quick Latex v2.6.5
		6: Iconize v2.11.4
		7: Editor Syntax Highlight v0.1.3
		8: Annotator v0.2.11
		9: Better Word Count v0.10.1
		10: Omnisearch v1.21.1
		11: Text Extractor v0.5.1
		12: Sortable v0.3.1
		13: Tab Switcher v1.4.0
		14: Calendar v1.5.10
		15: Version History Diff v2.1.12
		16: Git v2.24.1
		17: Smart Second Brain v0.6.0

**Comment by nicobrauchtgit** [comment]
Have you tried using Ollama locally? I also use a remote instance and it works for me so it might be an os specific issue. I have to check on a different os

**Comment by Hempflingclub** [comment]
yeah ollama locally poses no issues. But it doesnt really solve the issue.

**Comment by Leo310** [comment]
I know Its been some time, but it will be fixed in the next Release ;)
Wont take long anymore!
